scripts/dev/newReleaseBranch.py

```diff
@@ -0,0 +1,604 @@
+#######################################
+# This script creates HotFix branches for repo, share and packaging projects, and updates the master branches ready for the next SP/major release.
+# Run the script without passing any arguments or with -h/--help argument to get usage information.
+# When the script is run with indicating that next version is a major bump (-r/--release version major is lowera than -n/--next_dev version major)
+# then HF and SP branches get created. Otherwise (no major version bump), only HF branches are created.
+# Script can also be run ahead of release (-a/--ahead argument) and then it will only create release/X.Y.Z branches
+# See below script behaviour explained.
+#######################################
+# Create a HotFix branches for the released version (for X.Y.Z release it will be release/X.Y.N eg., create release/23.2.N for 23.2.0 release)
+# 1. acs-packaging:
+# - set RELEASE_VERSION to X.Y.1, DEVELOPMENT_VERSION to X.Y.2-SNAPSHOT in master_release.yml
+# - set POM versions to X.Y.1-SNAPSHOT
+# - set scm-tag in main POM to HEAD
+# 2. enterprise-share
+# - set scm-tag in main POM to HEAD
+# - set ACS version properties in main POM to X.Y.1
+# - set POM versions to  X.Y.1.1-SNAPSHOT
+# 3. enterprise-repo:
+# - set POM versions to  X.Y.1.1-SNAPSHOT
+# - set scm-tag in main POM to HEAD
+# - set acs.version.label to .1
+# 4. community-share
+# - set scm-tag in main POM to HEAD
+# - set ACS version properties in main POM to X.Y.1
+# - set main POM versions to  X.Y.1.1-SNAPSHOT
+# 5. community-repo:
+# - set ACS version properties in main POM to X.Y.1
+# - set POM versions to  X.Y.1.1-SNAPSHOT
+# - set scm-tag in main POM to HEAD
+# - increment schema by 1 in repository.properties
+# - set version.revision to 1 in version.properties (test resources)
+# 6. acs-community-packaging
+# - not created as we do not release hot fixes for community version
+#######################################
+# Create a HotFix branches for the released version (for X.Y.Z release it will be release/X.Y+1.0 eg., create release/23.3.0 for 23.2.0 release)
```

Comment:
```suggestion
# Create a Service Pack branches for the released version (for X.Y.Z release it will be release/X.Y+1.0 eg., create release/23.3.0 for 23.2.0 release)
```